flat_hash_map.hpp

```diff
@@ -4,6 +4,24 @@
 
 #pragma once
 
+// Check windows
+#if _WIN32 || _WIN64
```

Comment:
```suggestion
#if SIZE_MAX == UINT64_MAX
```

same goes for every other place